src/examples/component-definition/xml/example-component-definition.xml

```diff
@@ -46,17 +49,52 @@
         source="https://github.com/usnistgov/oscal-content/blob/master/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_MODERATE-baseline_profile.xml">
       <description>
         <p>MongoDB control implementations for NIST SP 800-53 revision 5.</p>
-      </description>
+      </description>          
+      <implemented-requirement
+          uuid="cf8338c5-fb6e-4593-a4a8-b3c4946ee080"
+          control-id="sc-8">
+        <description>
+          <p>MongoDB's implementation of SC-8 control. The details of the implementation are provided at the statement level.</p>
+        </description>
+        <set-parameter param-id="sc-8_prm_1">
+            <value>confidentiality</value>
+        </set-parameter>
+        <statement 
+          statement-id="sc-8_smt" 
+          uuid="bb9219b1-e51c-4680-abb0-616a43bbfbb1">      
+            <description>
+              <p>MongoDB implements TLS 1.x to protect the 
+                <insert type="param" id-ref="sc-8_prm_1"/> 
+              of transmitted data by encrypting data in transit,
+              preventing unauthorized disclosure or changes to information
+              during transmission.</p>
+            </description>
```

Comment:
Minor, non-blocking: One of us had recommended that we move to not manually create newlines in XML `text()` nodes when multi-line like the `description` and we used the Word Wrap feature of our IDEs instead. In the future, it will lead to clean diffing, can we change that before merge?

```suggestion
            <description>
              <p>MongoDB implements TLS 1.x to protect the <insert type="param" id-ref="sc-8_prm_1"/> of transmitted data by encrypting data in transit, preventing unauthorized disclosure or changes to information during transmission.</p>
            </description>
```